x/datapool/genesis.go

```diff
@@ -23,6 +23,13 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 		k.SetPool(ctx, &pool)
 	}
 
+	k.SetInstantRevenueDistribute(ctx, &genState.InstantRevenueDistribute)
+
+	for _, history := range genState.SalesHistories {
+		key := types.GetKeyPrefixSalesHistory(history.PoolId, history.Round, history.SellerAddress)
+		k.SetSalesHistoryByKey(ctx, key, history)
```

Comment:
I think you can use `k.SetSalesHistory()` itself.
```suggestion
		k.SetSalesHistory(ctx, history)
```

Reply:
Thank you!